index.html

```diff
@@ -77,33 +78,33 @@ <h5 class='offcanvas-title' id='offcanvasExampleLabel'>Menu</h5>
         <div class='offcanvas-body'>
           <ul class="offcanvas-menu  " >
             <li class="" >
-              <a class="btn btn-secundary" href="">
-                  <button class="btn btn-primary">About</button>     
+              <a class="" type="button" href="./views/about.html">
+                  <button class="btn btn-primary" type="button">About</button>     
               </a>                         
             </li>            
             <li class="" >
-              <a class="btn btn-secundary" href="">
-                  <button class="btn btn-primary">Program</button>     
+              <a class="" type="button" href="">
+                  <button class="btn btn-primary" type="button">Program</button>     
               </a>                         
```

Comment:
- [ ] Please the navbar should be divided into a top dark bar and a bottom white bar.
1. The top short bar section should be with social media links, log-out, and language links aligned to the right. as in the design.
Your design 
![image](https://user-images.githubusercontent.com/74744787/226133634-02667721-4b21-45a3-a892-34c3359da966.png)
Behance 
![image](https://user-images.githubusercontent.com/74744787/226133668-e26309de-0f6c-4593-beda-077ac1de08d6.png)

Reply:
done